packages/roosterjs-editor-plugins/lib/plugins/Paste/excelConverter/convertPastedContentFromExcel.ts

```diff
@@ -19,6 +19,15 @@ export default function convertPastedContentFromExcel(
     const { fragment, sanitizingOption, htmlBefore, clipboardData } = event;
     const html = excelHandler(clipboardData.html, htmlBefore);
 
+    // For Excel Online
+    if (fragment.childNodes.length > 0 && getTagOfNode(fragment.firstChild) == 'DIV') {
+        fragment.firstChild.childNodes.forEach(child => {
+            if (getTagOfNode(child) == 'TABLE') {
+                event.fragment.replaceChildren(child);
+            }
+        });
+    }
+
     if (clipboardData.html != html) {
         const doc = new DOMParser().parseFromString(trustedHTMLHandler(html), 'text/html');
         moveChildNodes(fragment, doc?.body);
```

Comment:
Actually this line will overwrite what you have done from line 22. Is that really ok?